wiki/en/Server-Linux.md

```diff
@@ -29,9 +29,9 @@ You can control Jamulus with the `systemctl` command. For example, to stop the s
 
 `sudo systemctl stop jamulus-headless`
 
-### Running in public mode
+### Running in Registered mode
 
-The following minimum setup is required to [run a public server](Running-a-Server#server-types):
+The following minimum setup is required to [run a Registered Server](Running-a-Server#server-types):
 
 ~~~
```

Comment:
I know it's unchanged... can we add a reference after line 39 (well, after line 55, I guess) to where the country IDs are specified?  It should also say clearly "Replace `genreServer:port` with one of the following options", rather than 44.